docs/app/docs/guides/pinning_packages.md

```diff
@@ -59,6 +59,39 @@ For example, to use the latest version of `ripgrep,` run `devbox add ripgrep`, `
 
 Whenever you run `devbox update`, packages with the latest tag will be updated to the latest version available in our index.
 
+### Restricting Packages to Specific Platforms
```

Comment:
🤔 I don't understand why this is in the "pinning packages" section. Isn't that about restricting to a certain version? 

Not totally sure this feature applies to this guide...

Reply:
I guess I was trying to find a guide to place it in, but not sure where the best place was. It seemed a little narrow for it's own page, but maybe that's the best place